solidity/random-beacon/hardhat.config.ts

```diff
@@ -133,21 +131,23 @@ const config: HardhatUserConfig = {
   typechain: {
     outDir: "typechain",
   },
+  gasReporter: {
+    currency: "USD",
+    coinmarketcap: process.env.COINMARKETCAP_API_KEY,
+  },
 }
 
-task(TASK_TEST, "Runs mocha tests").setAction(async (args, hre, runSuper) => {
+task("check-accounts-count", "Checks accounts count").setAction(async () => {
   // eslint-disable-next-line @typescript-eslint/no-var-requires,global-require
   const { constants } = require("./test/fixtures")
 
-  if (testConfig.operatorsCount < constants.groupSize) {
+  if (testConfig.operatorsCount < constants.groupSize4) {
```

Comment:
```suggestion
  if (testConfig.operatorsCount < constants.groupSize) {
```

Reply:
oops, fat finger fixed in [b7d2f6e](https://github.com/keep-network/keep-core/pull/2958/commits/b7d2f6e3a86b5b3161deab98a8a2cb33c599b0f2)